.github/workflows/lint.yml

```diff
@@ -8,14 +8,14 @@ on:
       - stable*
 
 jobs:
-  php:
+  php-lint:
     runs-on: ubuntu-latest
 
     strategy:
       matrix:
-        php-versions: ['7.2', '7.3', '7.4']
+        php-versions: ['7.3', '7.4', '8']
   
-    name: php${{ matrix.php-versions }}
+    name: Lint php${{ matrix.php-versions }}
```

Comment:
It changes all tests names, please put it back to `php${{ matrix.php-versions }}` :)

Reply:
Right, that was the goal behind. 😉 
`Lint php7.4` just tells you what it does and not only on which platform it operates. `php7.4` could be anything between phpunit, cs, linting...

Reply:
Yes, but the actions are displayed with the workflow name:
This was already done on purpose
![image](https://user-images.githubusercontent.com/14975046/113439213-6a4d1900-93ea-11eb-8305-b3a7acb7cd2f.png)

Reply:
Also, we use the same syntax all over nextcloud, so I'd prefer if we are consistent and so I don't have to change the protection config in multiple branches as well (github is awful for that)

Reply:
Didn't show here like that as i started iirc. Well anyways. 🤷‍♂️ 
![grafik](https://user-images.githubusercontent.com/47433654/113491532-bd4dcb80-94d1-11eb-9e75-705e94dc58b6.png)